testing/deployer/topology/topology.go

```diff
@@ -227,6 +277,10 @@ type Cluster struct {
 	//
 	// Denormalized during compile.
 	Peerings map[string]*PeerCluster `json:",omitempty"`
+
+	// EnableV2 activates V2 on the servers. If any node in the cluster needs
+	// V2 this will be turned on automatically.
+	EnableV2 bool `json:",omitempty"`
```

Comment:
Should this field be private then? The user wouldn't set it explicitly right?

Reply:
Like many other fields on the `topology` structs, this is treated as a derived/computed field, but left exported because it is consume outside of this package.